GO_SOPT_Seminar_Assignment/Local/TVING/Classes & Structure/ContentsSectionLayout.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  MainViewSection.swift
+//  GO_SOPT_Seminar_Assignment
+//
+//  Created by 김다예 on 2023/05/01.
+//
+
+import UIKit
+
+@frozen
+enum ContentsSectionLayout: String {
+    
+    case header = "Header"
+    case normal = "Normal"
+    case live = "Live"
+    case banner = "Banner"
+    
+    var itemSize: NSCollectionLayoutSize {
+        switch self {
+        case .header:
+            return NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0) ,heightDimension: .absolute(520))
+        case .normal:
+            return NSCollectionLayoutSize(widthDimension: .absolute(98) ,heightDimension: .absolute(160))
+        case .live:
+            return NSCollectionLayoutSize(widthDimension: .absolute(160) ,heightDimension: .absolute(140))
+        case .banner:
+            return NSCollectionLayoutSize(widthDimension: .absolute(300) ,heightDimension: .absolute(100))
+        }
+    }
+    
+    var interGroupSpacing: CGFloat {
+        switch self {
+        case .header:
+            return 0
+        default:
+            return 10
+        }
+    }
```

Comment:
움 .header 를 제외한 다른 모든 경우라 default 를 쓰신 것 같은데
가독성을 위해 아래와 같이 하면 어떨까요??!

```suggestion
    var interGroupSpacing: CGFloat {
        switch self {
        case .header:
            return 0
        case .normal, .live, .banner:
            return 10
        }
    }
```

완전 개인 취향인 것 같아서 편하신대로 해도 될 것 같긴 해요 :)